xbmc/pvr/PVRContextMenus.cpp

```diff
@@ -19,7 +19,6 @@
  */
 
 #include "ContextMenuItem.h"
-#include "cores/AudioEngine/Engines/ActiveAE/AudioDSPAddons/ActiveAEDSP.h"
```

Comment:
`ServiceBroker.h` need to be included by this file
it was previously included through `ActiveAEDSP.h`